.github/workflows/test.yml

```diff
@@ -0,0 +1,56 @@
+name: Test
+on:
+  push:
+  pull_request:
+
+jobs:
+  test:
+    name: Test
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
+        go:
+          - "1.x"
+
+    steps:
+      - name: Set up Go ${{ matrix.go }}
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go }}
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+
+      - name: restore cache
+        uses: actions/cache@v2
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+
+      - name: Test
+        run: make cover
+        env:
+          GITHUB_TOKEN: ${{ secrets.GHTOOLS_GITHUB_TOKEN }}
```

Comment:
Please set secrets for GitHub Actions.
https://docs.github.com/en/actions/reference/encrypted-secrets
https://docs.github.com/en/rest/reference/actions#secrets

It needs collaborator access to do it.